src/main/java/bio/terra/service/snapshotbuilder/utils/AggregateBQQueryResultsUtils.java

```diff
@@ -15,7 +15,7 @@ public static List<SnapshotBuilderConcept> aggregateConceptResults(TableResult r
                     .id((int) (row.get("concept_id").getLongValue()))
                     .name(row.get("concept_name").getStringValue())
                     .hasChildren(true)
-                    .count(1))
+                    .count((int) (row.get("count").getLongValue())))
```

Comment:
This code is used by both `getConceptChildren` and `searchConcepts`. I think this will break `getConceptChildren` unless you make this conditional, or if you update `getConceptChildren` to also generate a `count` column.